charts/datahub/values.yaml

```diff
@@ -62,6 +62,10 @@ global:
   datahub_analytics_enabled: true
   datahub_standalone_consumers_enabled: false
 
+
+  globalIngress:
```

Comment:
Any reason we've picked `globalIngress` here and not just `ingress`? It's already under the `global` section so it just feels more verbose.